docs/source/federated-types/federated-directives.mdx

```diff
@@ -577,6 +577,9 @@ extend schema
     # highlight-start
     @composeDirective(name: "@myDirective")
     @composeDirective(name: "@hello")
+
+directive @myDirective(a: String!) on FIELD_DEFINITION
```

Comment:
I can't tell for sure with the diff but could we make sure all these lines are indented consistently?